addons/monitoring/manifests/plutono/configmap.yaml

```diff
@@ -30,8 +30,8 @@ data:
     provisioning = /etc/plutono/provisioning
     [server]
     # these settings needed to make it work with Ingress
-    root_url = https://k2s-monitoring.local
-    domain = k2s-monitoring.local
+    root_url = https://k2s-monitoring.local/monitoring
+    domain = k2s.cluster.net
```

Comment:
I really like the new approach with `https://k2s.cluster.net/monitoring` but this will not work anymore now with the old one `k2s-monitoring.local`.

Reply:
I also wondered why, but it DOES work - I manually tested all combinations, and I also e2e test them.
But I will double-check once more!

Yes, we could deprecate the old approach, of using the hostnames for the ingress path matches.
On the other hand, these addons serve as "how to" for product developers, and I believe it is also a cool use case - sometimes you want this and sometimes that.

I want us to think more on when and what to remove - one solution would be to keep e.g. the k2s-dashboard.local, and move all others to the central cluster ingress...

Reply:
We should keep the possibility to have specific host names for each app. The default shall be k2s.cluster.net. I wonder why it does not work in your case Joachim ?

Reply:
I haven't tested it (will test it today) but now there is a mix of both - `https://k2s-monitoring.local/monitoring` should be `https://k2s.cluster.net/monitoring` right?

Reply:
absolutely, it is inconsistent and I'll correct it.